src/databricks/sqlalchemy/requirements.py

```diff
@@ -1,18 +0,0 @@
-# Following official SQLAlchemy guide:
```

Comment:
where did this class come from? do we already have some sql-alchemy related code in the main branch?

Reply:
This was created initially from the rough scaffolding that Jesse and I put together earlier on (commit 0378460e0aaa272f6a38193d47c5289cc5d2b98a). I ended up tossing it after I figured out that I won't be using SA's test suite.